src/fin/test/gcd_fill.tcl

```diff
@@ -0,0 +1,10 @@
+read_db gcd_fill.odb
+read_sdc gcd_fill.sdc
+
+set_propagated_clock [all_clocks]
```

Comment:
same - timing not needed

Reply:
You are right, we don't need the timing. I was just doing copy paste from flow scripts to make a test case. Will remove.

Reply:
And therefore we  don't need the sdc either.